taxi/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.urls import path
+
+from taxi import views
+
```

Comment:
Consider adding a blank line between the import statements and the `urlpatterns` list for better readability and to adhere to PEP 8 style guidelines.